### PR TITLE
portal(dm): delete protal code

### PR DESCRIPTION
**Pull request by Ehco1996**
### What problem does this PR solve?

Issue Number: ref #3583

### What is changed and how it works?

only DM v1.0 support dm-portal, so we can delete those dead code now

### Check List 

Tests 

 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Breaking backward compatibility

Related changes

### Release note 

```release-note
`None`.
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

**Comment by okJiang** [comment]
need rebase @Ehco1996

**Comment by Ehco1996** [comment]
> need rebase @Ehco1996

done ~~

**Review by glorv — APPROVED** [review]

File: docs/swagger/swagger.json [hunk 1]
```diff
@@ -1,998 +1,902 @@
 {
```
Comment (okJiang) [inline docs/swagger/swagger.json:1]: what does this file do?

**Comment by Ehco1996** [comment]
/hold 
should fix wrong merge master

File: docs/swagger/swagger.json (hunk 1 above)
Comment (Ehco1996) [reply docs/swagger/swagger.json]: fixed~

**Review by okJiang — APPROVED** [review]

**Pull request merged**